src/sentry/static/sentry/app/views/organizationEventsV2/data.jsx

```diff
@@ -64,6 +64,28 @@ export const ALL_VIEWS = deepFreeze([
     ],
     columnWidths: ['3fr', '70px', '70px', '1fr', '1.5fr'],
   },
+  {
+    id: 'transactions',
+    name: 'Transactions',
+    data: {
+      fields: ['event', 'type', 'project', 'user', 'time'],
+      orderby: ['-timestamp', '-id'],
+      query: 'event.type:transaction',
+    },
+    tags: [
+      'event.type',
+      'release',
+      'project.name',
+      'user.email',
+      'user.ip',
+      'environment',
+      'trace',
+      'trace.ctx',
+      'trace.span',
```

Comment:
I think we will have too many values in the trace related tags to do any useful navigation with them.

Reply:
I agree. I'm wondering if there's any value to this particular view, since transaction events are not grouped by their trace IDs 🤔

Reply:
Probably not.

Reply:
This will be addressed in https://github.com/getsentry/sentry/pull/14103